neo.UnitTests/UT_GasToken.cs

```diff
@@ -94,6 +97,37 @@ public void Check_BalanceOfAndTransfer()
 
             snapshot.Storages.GetChangeSet().Count().Should().Be(keyCount); // All
 
+            // Burn
+
+            var engine = new ApplicationEngine(TriggerType.Application, null, snapshot, 0);
+            keyCount = snapshot.Storages.GetChangeSet().Count();
+
+            Assert.ThrowsException<TargetInvocationException>(() =>
+                typeof(GasToken).GetMethod("Burn", BindingFlags.NonPublic | BindingFlags.Instance)
+                .Invoke(NativeContract.GAS, new object[] { engine, new UInt160(to), BigInteger.MinusOne }));
+
+            // Burn more than expected
+
+            Assert.ThrowsException<TargetInvocationException>(() =>
+                typeof(GasToken).GetMethod("Burn", BindingFlags.NonPublic | BindingFlags.Instance)
+                .Invoke(NativeContract.GAS, new object[] { engine, new UInt160(to), new BigInteger(800000000001) }));
+
+            // Real burn
+
+            typeof(GasToken).GetMethod("Burn", BindingFlags.NonPublic | BindingFlags.Instance)
```

Comment:
You can call `GasToken.Burn()` directly without reflection.

Reply:
Done!